lib/streamlit/web/server/server.py

```diff
@@ -92,6 +92,8 @@
 UNIX_SOCKET_PREFIX: Final = "unix://"
 
 MEDIA_ENDPOINT: Final = "/media"
+COMPONENT_ENDPOINT: Final = "component"
+STATIC_SERVING_ENDPOINT: Final = "app/static"
 UPLOAD_FILE_ENDPOINT: Final = "/_stcore/upload_file"
 STREAM_ENDPOINT: Final = r"_stcore/stream"
 METRIC_ENDPOINT: Final = r"(?:st-metrics|_stcore/metrics)"
```

Comment:
Its a bit inconsistent where we use a slash in the beginning. Most endpoints here don't use a slash. Ultimately, there is most likely no difference... but something to unify at some point.